secrets/vault/vault_test.go

```diff
@@ -45,44 +45,28 @@ func (h *harness) MakeDriver(ctx context.Context) (driver.Keeper, driver.Keeper,
 	return &keeper{keyID: keyID1, client: h.client}, &keeper{keyID: keyID2, client: h.client}, nil
 }
 
-func (h *harness) Close() {
-	h.close()
-}
+func (h *harness) Close() {}
 
 func newHarness(ctx context.Context, t *testing.T) (drivertest.Harness, error) {
-	// Start a new test server.
-	c, cleanup := testVaultServer(t)
+	c, err := Dial(ctx, &Config{
+		Token: testToken,
+		APIConfig: api.Config{
+			Address: apiAddress,
+		},
+	})
+	if err != nil {
+		return nil, err
```

Comment:
If the local Vault server is not available, this should `Skip` the test instead of failing....

Reply:
See https://github.com/google/go-cloud/pull/1987 for an example.

Reply:
Dial doesn't make request, added `Skip` to the next call.